lib-injection/sources/min_compatible_versions.csv

```diff
@@ -159,15 +158,14 @@ ragas,==0.1.21
 redis,~=2.0
 redis-py-cluster,>=2.0
 reno,0
-requests,~=2.20.0
+requests,~=2.23.0

```

Comment:
🤔

Reply:
This matches the riotfile

Reply:
yeah, I think I was surprised it is changing, maybe since min compatible versions file is out of date?